src/test/cypress/e2e/exercises/quiz-exercise/QuizExerciseDropLocation.cy.ts

```diff
@@ -0,0 +1,60 @@
+import { Course } from 'app/entities/course.model';
+import { admin } from '../../../support/users';
+import { courseManagementRequest, quizExerciseDragAndDropQuiz } from '../../../support/artemis';
+import { convertCourseAfterMultiPart } from '../../../support/requests/CourseManagementRequests';
+import { MODELING_EDITOR_CANVAS } from '../../../support/pageobjects/exercises/modeling/ModelingEditor';
+
+let course: Course;
+
+describe('Quiz Exercise Drop Location Spec', () => {
+    before('Set up course', () => {
+        cy.login(admin);
+        courseManagementRequest.createCourse().then((response) => {
+            course = convertCourseAfterMultiPart(response);
+        });
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequest.deleteCourse(course.id!);
+    });
+
+    describe('DnD Quiz drop locations', () => {
+        before('Create DND quiz', () => {
+            cy.login(admin, '/course-management/' + course.id + '/exercises');
+            cy.get('#create-dd-quiz').should('be.visible').click();
+            cy.get('#create-apollon-diagram').should('be.visible').click();
+            cy.get('#field_title').type('cypress-dd-quiz');
+            cy.get('#save-dd-quiz').click();
+        });
+        it('Checks drop locations', () => {
+            let containerBounds: DOMRect;
+            quizExerciseDragAndDropQuiz.dragUsingCoordinates(310, 320);
+            quizExerciseDragAndDropQuiz.dragUsingCoordinates(730, 500);
+            quizExerciseDragAndDropQuiz.dragUsingCoordinates(1000, 100);
+            cy.get('#modeling-editor-sidebar').children().eq(0).children().eq(1).click();
```

Comment:
Without looking at the actual dom element, I could not understand what is clicked here. do you mind putting the children's elements into a local variable to make it more readable? or putting this statement into a separate method may also be helpful as well.